bin/bug_xxx_newtab.py

```diff
@@ -59,14 +59,15 @@ def migrate(ctx):
 newtab-settings-button =
     .title = { COPY(from_path, "settings_pane_button_label") }
 
-newtab-search-box-search-text = { COPY(from_path, "search_button") }
 newtab-search-box-search-button =
     .title = { COPY(from_path, "search_button") }
+    .aria-label = { COPY(from_path, "search_button") }
```

Comment:
In general, should we always be putting aria-label for anything that has a title attribute? At minimum it's the same string, but potentially aria-label can be different?

Reply:
E.g., newtab-settings-button a few lines above has just `.title`

Reply:
Yep! If they're duplicated, we make sure that screen readers are all getting the same information no matter which prop they use, especially if there's no anchor text. The aria-label could potentially be different, like in a case where it's describing special behaviour (I think that'll be the case for the drag-and-drop of the Top Sites, when I build an accessible container for them.)